al/auxeffectslot.h

```diff
@@ -91,6 +91,7 @@ struct ALeffectslot {
 
 private:
     ALCcontext* eax_al_context_{};
+    const EaxCall* eax_call_{};
```

Comment:
I don't think I like this being something stored in the object, which is updated before an `eax_dispatch` call. It makes it harder to reason about what it's actually doing, since the information is filled in and stored elsewhere. Especially as a pointer to const, which indicates the object shouldn't be getting modified.

Reply:
Fixed.